### ISSUE TITLE
Phpbb 3.0.14 / Api id/verfication code = The value you entered is too short.

### ISSUE THREAD

**Issue by psailleko** [shibdib/API-Registration-Mod-Revisited#8]
Hi,

 i've just installed your mod with automod on a fresh phpbb 3.0.14,
when i want to add an api key/code to an existing or a new user on the forum i get the message 

The value you entered is too short.
The value you entered is too short.

 Does it ring a bell for you ? 
I've probably missed a step :x

**Comment by shibdib** [comment]
it should work, is it an old api key with a super short keyid?

**Comment by psailleko** [comment]
Nope, it's a fresh new api key with 64 characters for the validation key and 7 for the key id.

 I've noticed that i cannot get the Eve modules in the general menu, i need to add them as new tabs in the acp .. doesn't it means my install if borked ?

**Comment by vedalken254** [comment]
Shib: I've got that issue too. There's no schema for changes to the users table that I've seen in 7.0.7 or here on github. Could you please include a database schema for those changes or tell me what the appropriate columns and column settings are?
